src/game/server/instagib/round_stats.cpp

```diff
@@ -123,41 +123,61 @@ void IGameController::GetRoundEndStatsStrPsv(char *pBuf, size_t Size)
 	int ScoreLimit = m_GameInfo.m_ScoreLimit;
 	int TimeLimit = m_GameInfo.m_TimeLimit;
 
-	std::optional<std::string> RedClan = std::nullopt;
-	std::optional<std::string> BlueClan = std::nullopt;
+	char *RedClan = nullptr;
+	char *BlueClan = nullptr;
```

Comment:
This should be `const char *pRedClan = nullptr;`

Reply:
If this will be const, how could I change it later

Reply:
Yea thats a bit cursed haha. But you don't want to change the clan. A const pointer can point to memory and promises it won't change the memory but you can still point the pointer to something else after its assigned.

This is ok

```C++
const char *pRedClan = nullptr;
pRedClan = Server()->ClientClan(pPlayer->GetCid());
```

This is not

```C++
const char *pRedClan = nullptr;
pRedClan = Server()->ClientClan(pPlayer->GetCid());
pRedClan[0] = 'x'; // we can not change the clan 
```

Reply:
Interesting, I thought const in this case indicates const pointer, like it's pointing to one address only